install/lighttpd/install.yml

```
@@ -0,0 +1,15 @@
+id: lighttpd
+name: lighttpd
+title: Lighttpd
+description: |
+  To collect data and deliver it to New Relic, integrate your Lighttpd processes with our Infrastructure agent and Flex configuration.
```

```suggestion
  To collect data and deliver it to New Relic, integrate your lighttpd processes with our Infrastructure agent and Flex configuration.
```

Based on its website, lighttpd is always lowecase

---

Also, did you know lighttpd is pronounced _lighty_? :)

---

Yes, Shawn.

---

I didn't :) I was very surprised!